rpcserver.go

```diff
@@ -574,7 +597,7 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 		}
 	}
 
-	if specificGroupInternalKey {
+	if req.Asset.GroupInternalKey != nil {
```

Comment:
This doesn't have the same meaning as before. `specificGroupInternalKey` was `true` if the seedling is being minted into an existing group.

`GroupInternalKey` is only non-nil if you are creating a _new_ group, _and_ you want to use a specific internal key instead of whichever key `tapd` will generate automatically.